app/models/user.rb

```diff
@@ -1,3 +1,9 @@
 class User < ApplicationRecord
-  has_many :posts, :comments, :likes
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
```suggestion
  has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

- [ ] Kindly make sure that user models have a foreign key with author_id for posts, comments and likes 👍🏿